include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -31,6 +31,7 @@ typedef struct _oe_sgx_enclave_image_info_t
 // oe_sgx_enclave_properties_t SGX enclave properties derived type
 #define OE_SGX_FLAGS_DEBUG 0x0000000000000002ULL
 #define OE_SGX_FLAGS_MODE64BIT 0x0000000000000004ULL
+#define OE_SGX_FLAGS_KSS 0x0000000000000008ULL
```

Comment:
This has to match SGX_FLAGS_KSS which is

`#define SGX_FLAGS_KSS            0x0000000000000080ULL     /* If set enclave uses KSS */`

Reply:
Sure, I will reset it to SGX_FLAGS_KSS value. What is the context we have OE_SGX_FLAGS_xxx so have to maintain two set of flag values?

Reply:
> Sure, I will reset it to SGX_FLAGS_KSS value. What is the context we have OE_SGX_FLAGS_xxx so have to maintain two set of flag values?

@CodeMonkeyLeet Why are we keeping 2 sets of similar definitions?